packages/vscode-extension/src/webview/components/Preview.tsx

```diff
@@ -426,7 +431,10 @@ function Preview({
       }
     }
     addEventListener("blur", onBlurChange, true);
-    return () => removeEventListener("blur", onBlurChange, true);
+    return () => {
+      window.removeEventListener("contextmenu", onContextMenu);
+      removeEventListener("blur", onBlurChange, true);
```

Comment:
can we update `removeEventListener` here and also corresponding `addEventLIstener` call to specify whether it operates on document or window (or something else?)